{"name": "test_delete_added_request", "status": "broken", "statusDetails": {"message": "AttributeError: 'list' object has no attribute 'find_elements_by_tag_name'", "trace": "self = <test_case_details.TestCaseDetails object at 0x0DBDC850>\n\n    def test_delete_added_request(self):\n        driver = self.driver\n        self.login()\n        case_details = CaseDetails(driver)\n        home = HomePage(driver)\n        home.click_law_suit_page()\n        case_details.request_tab_entry()\n>       case_details.delete_added_request()\n\ntest_case_details.py:87: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <Pages.CaseDetailsPage.CaseDetails object at 0x0DBDCAD0>\n\n    def delete_added_request(self):\n        table = self.driver.find_elements_by_xpath(self.directive_table_xpath)\n    \n        for items in table:\n            trs = items.find_elements_by_tag_name(\"tr\")\n>           tds = trs.find_elements_by_tag_name(\"td\")\nE           AttributeError: 'list' object has no attribute 'find_elements_by_tag_name'\n\n..\\Pages\\CaseDetailsPage.py:179: AttributeError"}, "start": 1568619871243, "stop": 1568619888873, "uuid": "0cfa6861-87e0-4541-806f-046f649bd906", "historyId": "812199f097f3799c533dced022272d50", "testCaseId": "546de6c208ed1c45d32a4ab8f8a947c2", "fullName": "test_case_details.TestCaseDetails#test_delete_added_request", "labels": [{"name": "parentSuite"}, {"name": "suite", "value": "test_case_details"}, {"name": "subSuite", "value": "TestCaseDetails"}, {"name": "host", "value": "EVT01066NB"}, {"name": "thread", "value": "1492-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_case_details"}]}